book/providers/ipc.md

```diff
@@ -1,5 +1,82 @@
 # IPC provider
+The IPC (Inter-Process Communication) transport is a way for a process to communicate with a running Ethereum client over a local Unix domain socket. If you are new to IPC, you can [follow this link to learn more](https://en.wikipedia.org/wiki/Inter-process_communication). Using the IPC transport allows the ethers library to send JSON-RPC requests to the Ethereum client and receive responses, without the need for a network connection or HTTP server. This can be useful for interacting with a local Ethereum node that is running on the same machine. Using Ipc [is faster than RPC](https://github.com/0xKitsune/geth-ipc-rpc-bench), however you will need to have a local node that you can connect to.
```

Comment:
Can you also add a note on [Windows IPC (Named pipes)](https://learn.microsoft.com/en-us/windows/win32/ipc/named-pipes), recently added in #1976? 
The only API difference is the path: `\\<machine_address>\pipe\<pipe_name>`, for a local geth: `\\.\pipe\geth`
Thanks ^^

Reply:
No problem, will add this in shortly.

Reply:
I just added a note about IPC on windows in the most recent push, let me know if you think there should be any changes.